bower.json

```diff
@@ -12,6 +12,6 @@
     "tests"
   ],
   "dependencies": {
-      "test_track_js_client": "https://github.com/Betterment/test_track_js_client.git#v1.3.3"
+      "test_track_js_client": "https://github.com/Betterment/test_track_js_client.git#v1.3.6"
```

Comment:
We can swap this to pull from bower, right? Or, we still have to publish the js client to bower. I will add that as an issue on the js repo.